communication/slack-config/channels.yaml

```diff
@@ -237,6 +237,7 @@ channels:
   - name: rancher
   - name: random
     archived: true
+  - name: release-ci-signal
```

Comment:
It should be added to [sig-release/config.yaml](https://github.com/kubernetes/community/blob/master/communication/slack-config/sig-release/config.yaml) instead of the main config. 

Both `sig-release-*` and `release-*` have been delegated so the sig-release leads can approve 👍 
https://github.com/kubernetes/community/blob/33030c8fc0a247d360b5f55c7d63b603f02b6084/communication/slack-config/restrictions.yaml#L16-L19

Reply:
@mrbobbytables -- good catch! I forgot about that!

Reply:
Thank you! I wasn't aware of this